packages/loopring_v3/contracts/amm/libamm/AmmUpdateProcess.sol

```diff
@@ -72,6 +77,13 @@ library AmmUpdateProcess
             txsDataPtr += ExchangeData.TX_DATA_AVAILABILITY_SIZE;
         }
 
+        if (start && ctx.settings.assetManager != IAssetManager(0)) {
```

Comment:
what if `assetManager` is non-zero, then `S.balancesL1[...] ` is non-zero, then we set `assetManager` to zero, now `S.balancesL1[...] ` will still be non-zero, shall we still do the `add` operation？

Reply:
It's currently impossible to update the `assetManager` address in the pool, it's an `immutable` value hardcoded in the implementation contract.

If we would allow setting the `assetManager` back to zero, I would say we should make that impossible if `balancesL1` are non-zero.

Reply:
Keep it as immutable is fine.